client/src/components/Header/Header.js

```diff
@@ -12,30 +13,118 @@ export default class Header extends React.Component {
+                            <img className='header-icon' src={process.env.PUBLIC_URL + '/MM-logo-rev-3.png'} alt="my image" style={{objectFit: 'cover'}}/>
+                        </div>
+                        <div className="tabletCenter">
+                            <Modal show={this.state.isSearchModalOpen} onHide={() => {this.setState({isSearchModalOpen: false})}} animation={false}>
+                                <Modal.Header closeButton>
+                                    <img className='header-icon' src={process.env.PUBLIC_URL + '/MM-logo-rev-3.png'} alt="my image" width='35px' height='35px' style={{marginBottom: '1%'}}/>
+                                </Modal.Header>
+                                <SearchBar barWidth="100%" locationSearchMargin='0%' buttonWidth='100%' barBottomSpacing='1%' closeModal={() => {this.setState({isSearchModalOpen: false})}}/>
+                            </Modal>
+                        </div>
+                        <div className="tabletRight">
+                            <Button variant="link-secondary" style={{padding: 0, marginRight: '5%'}}><img className='header-icon' src={process.env.PUBLIC_URL + '/search-512.png'} alt="my image" onClick={() => {this.setState({isSearchModalOpen: true})}} /></Button>
+                            <Button variant="link-secondary" style={{padding: 0}}><img className='header-icon' src={process.env.PUBLIC_URL + '/menu-alt-512.png'} alt="my image" onClick={() => {this.setState({isMenuOpen: true})}} /></Button>
+                            <CheeseburgerMenu isOpen={this.state.isMenuOpen} closeCallback={() => {this.setState({isMenuOpen: false})}}>
+                                <div className="my-menu-content">
+                                    <ul>
+                                        <div className="links d-lg-block">
+                                            {this.links.map(link =>
+                                                <li><NavLink onClick={e => {
+                                                    e.preventDefault();
+                                                    window.location.replace(link.route);
+                                                }} to={link.route} exact={link.exact} className="header-link mr-4" activeClassName="active" key={link.name}>{link.name}</NavLink></li>
+                                            )}
+                                        </div>
+                                        <hr/>
+                                        <li><Button size="sm" variant="link-secondary" style={{padding: 0}}>Log in</Button></li>
+                                        <li><Button size="sm" variant="link-secondary" style={{padding: 0}}>Sign up</Button></li>
+                                    </ul>
+                                </div>
+                            </CheeseburgerMenu>
+
+                        </div>
+                    </>
+                }
```

Comment:
I'm not too familiar with mobile friendliness, but can't this be done via media breakpoints in `.scss` files?

Reply:
I believe we've done it that way for other pages.